### PR TITLE
DIV-6313: added Addressee, CtscContactDetails and LetterAddressHelper

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6313

### Change description ###
Reusable code

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/Addressee.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@EqualsAndHashCode
+public class Addressee {
+    @JsonProperty("name")
```
Comment (jvictorcf) [inline src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/Addressee.java:14]: I'd call it addressee name (unless the adressee class is an element in itself).

File: src/test/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetailsTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.bsp.common.model.document;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CtscContactDetailsTest {
+
+    private static final String CTSC_CARE_OF = "c/o HMCTS Digital Divorce";
```
Comment (jvictorcf) [inline src/test/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetailsTest.java:9]: I've commented this in one of John's PRs. I think the "c/o " bit should be in Docmosis, not here.
I've seen other templates that use this information, but without the "c/o"

File: src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/Addressee.java (hunk 1 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/Addressee.java:14]: yeah, the plan is to have: `addressee.name`

File: src/test/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetailsTest.java (hunk 2 above)
Comment (yarpo) [reply src/test/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetailsTest.java:9]: it's a mocked data for testing :)

**Pull request merged**